tests/Unit/Domain/CoordinateMaps/Test_CoordinateMap.cpp

```diff
@@ -1011,6 +1011,44 @@ void test_time_dependent_map() {
           .get(0, 0),
       (DataVector{2. / 3., 2. / 3., 2. / 3.}));
 }
+
+void test_push_back() {
+  INFO("Coordinate map with affine map");
+  using affine_map = CoordinateMaps::Affine;
+  using affine_map_2d = CoordinateMaps::ProductOf2Maps<affine_map, affine_map>;
+  using affine_map_3d =
+      CoordinateMaps::ProductOf3Maps<affine_map, affine_map, affine_map>;
+
+  const auto helper = [](const auto& first_map,
```

Comment:
maybe name this `check_coord_map_push` or similarly more informative lambda name instead of `helper`?

Reply:
Yea... I'm bad at that. Thanks for calling me out on it :)